encoder-otel-brave/src/test/java/zipkin2/reporter/otel/brave/JaegerAllInOne.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.reporter.otel.brave;
+
+import java.util.Collections;
+import java.util.Set;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+
+class JaegerAllInOne extends GenericContainer<JaegerAllInOne> {
```

Comment:
I assume there's no opentelemetry project that allows read-back of spans and that's why we are using jaeger. if so, comment that?

Reply:
This class is replaced by [ITOtelEncoderTest](https://github.com/openzipkin-contrib/zipkin-otel/pull/7/files#diff-f1614b313ae6e7ebe173556465333def6f893b421571522e9bb496a42ede83c3)